packages/contracts-bedrock/package.json

```diff
@@ -25,7 +25,7 @@
     "gas-snapshot": "pnpm build:go-ffi && pnpm gas-snapshot:no-build",
     "kontrol-summary": "./test/kontrol/scripts/make-summary-deployment.sh",
     "kontrol-summary-fp": "KONTROL_FP_DEPLOYMENT=true pnpm kontrol-summary",
-    "snapshots": "forge build && npx tsx scripts/autogen/generate-snapshots.ts && pnpm kontrol-summary && pnpm kontrol-summary-fp",
+    "snapshots": "forge build && npx tsx scripts/autogen/generate-snapshots.ts && pnpm kontrol-summary-fp && pnpm kontrol-summary",
```

Comment:
Just curious why the ordering changed here (it was regular summary then FP summary, now it's the opposite)?

Reply:
Because both summaries are written to the same JSON file (`31337.json`). Since the tests that are run are for the original OptimismPortal, we need to have the JSON file that doesn't deploy OptimismPortal2. Does this make sense?
I swaped it after seeing that when I recreated the summaries without the fault proofs, it changed the JSON file

Reply:
Sorry, I don't understand. Wouldn't the `31337.json` file get overwritten each time it's ran?

Reply:
Indeed. So, the committed `31337.json` has to contain the state updates for deploying the original OptimismPortal contract since these are the proofs currently run on the `develop` CI.
Now, the `shapshots:check` job will run both deployments and save the state updates of each into the `31337.json` file. But if the fault proofs deployment is last, the `snapshots:check` job will fail, since the last deployment written to `31337.json` is with the fault proofs, and not the original one (which is the one committed).
Let me know if I'm not making a mess of an explanation here!